### PR TITLE
Restructuring and formatting

### PR DESCRIPTION
**Pull request by hhsecond**
Restructure modules into multiple files and black formatting.

### CODE REVIEW

File: redisai/command_builder.py [hunk 1]
```diff
@@ -6,125 +6,151 @@
...
-            raise TypeError(f'``{dtype}`` is not supported by RedisAI. Currently '
-                            f'supported types are {list(utils.dtype_dict.keys())}')
+            raise TypeError(
+                f"``{dtype}`` is not supported by RedisAI. Currently "
+                f"supported types are {list(utils.dtype_dict.keys())}"
+            )
         except AttributeError:
-            raise TypeError("tensorset() missing argument 'dtype' or "
-                            "value of 'dtype' is None")
+            raise TypeError(
```
Comment (alonre24) [inline redisai/command_builder.py:93]: testable?

File: redisai/command_builder.py [hunk 2]
```diff
@@ -6,125 +6,151 @@
...
+                "tensorset() missing argument 'dtype' or value of 'dtype' is None"
+            )
         if shape is None:
             shape = (len(tensor),)
-        args = ['AI.TENSORSET', key, dtype, *shape, 'VALUES', *tensor]
+        args = ["AI.TENSORSET", key, dtype, *shape, "VALUES", *tensor]
     else:
-        raise TypeError(f"``tensor`` argument must be a numpy array or a list or a "
-                        f"tuple, but got {type(tensor)}")
+        raise TypeError(
```
Comment (alonre24) [inline redisai/command_builder.py:100]: testable?

File: redisai/dag.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        if load:
+            if not isinstance(load, (list, tuple)):
+                self.commands += ["LOAD", 1, load]
+            else:
+                self.commands += ["LOAD", len(load), *load]
+        if persist:
+            if not isinstance(persist, (list, tuple)):
+                self.commands += ["PERSIST", 1, persist, "|>"]
+            else:
+                self.commands += ["PERSIST", len(persist), *persist, "|>"]
```
Comment (alonre24) [inline redisai/dag.py:35]: Can we test "persist"?

**Review by alonre24 — APPROVED** [review]

File: redisai/dag.py (hunk 3 above)
Comment (hhsecond) [reply redisai/dag.py:35]: I see that we are already testing persist. I am merging this for now and will add more tests as needed

**Pull request merged**